README.md

```diff
@@ -1,2 +1,8 @@
-# nodejs-stub
-Stub of a Node.js project
+# Static Server
+
+Static server returning html file.
+Starts express server on host and port from configuration and configures it to provide _resources/pages/index.html_ file.
+
+* start - ``npm run start``
```

Comment:
It's not quite true. `npm run start` won't start a server.
Let's create `app.js` file that will create an instance of the Server and then start it.
Our `start` script will have to call this `app` file for execution.